R/weo_validate_raw.R

```diff
@@ -0,0 +1,150 @@
+                description = "`1999` should be character") |>
+    validate_if(is.character(`2000`),
+                description = "`2000` should be character") |>
+    validate_if(is.character(`2001`),
+                description = "`2001` should be character") |>
+    validate_if(is.character(`2002`),
+                description = "`2002` should be character") |>
+    validate_if(is.character(`2003`),
+                description = "`2003` should be character") |>
+    validate_if(is.character(`2004`),
+                description = "`2004` should be character") |>
+    validate_if(is.character(`2005`),
+                description = "`2005` should be character") |>
+    validate_if(is.character(`2006`),
+                description = "`2006` should be character") |>
+    validate_if(is.character(`2007`),
+                description = "`2007` should be character") |>
+    validate_if(is.character(`2008`),
+                description = "`2008` should be character") |>
+    validate_if(is.character(`2009`),
+                description = "`2009` should be character") |>
+    validate_if(is.character(`2010`),
+                description = "`2010` should be character") |>
+    validate_if(is.character(`2011`),
+                description = "`2011` should be character") |>
+    validate_if(is.character(`2012`),
+                description = "`2012` should be character") |>
+    validate_if(is.character(`2013`),
+                description = "`2013` should be character") |>
+    validate_if(is.character(`2014`),
+                description = "`2014` should be character") |>
+    validate_if(is.character(`2015`),
+                description = "`2015` should be character") |>
+    validate_if(is.character(`2016`),
+                description = "`2016` should be character") |>
+    validate_if(is.character(`2017`),
+                description = "`2017` should be character") |>
+    validate_if(is.character(`2018`),
+                description = "`2018` should be character") |>
+    validate_if(is.character(`2019`),
+                description = "`2019` should be character") |>
+    validate_if(is.character(`2020`),
+                description = "`2020` should be character") |>
+    validate_if(is.character(`2021`),
+                description = "`2021` should be character") |>
+    validate_if(is.character(`2022`),
+                description = "`2022` should be character") |>
+    validate_if(is.character(`2023`),
+                description = "`2023` should be character") |>
+    validate_if(is.character(`2024`),
+                description = "`2024` should be character") |>
+    validate_if(is.character(`2025`),
+                description = "`2025` should be character") |>
+    validate_if(is.character(`2026`),
+                description = "`2026` should be character") |>
+    validate_if(is.character(`2027`),
+                description = "`2027` should be character") |>
+    validate_if(is.character(`2028`),
```

Comment:
There should be an easier way to check all these columns. First, these columns will increase and we can't update the code each time they do. Second, the importance here is that they years exists, so that's what is needed to be checked. There should no be jumps in the years.

Reply:
Thanks for the comment, I have implemented your comment in the latest PR.